src/curl_jolokia.c

```diff
@@ -0,0 +1,1100 @@
+/**
+ * collectd - src/curl_jolokia.c
+ * Copyright (C) 2009       Doug MacEachern
+ * Copyright (C) 2006-2013  Florian octo Forster
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; only version 2 of the License is applicable.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+ *
+ * Authors:
+ *   Doug MacEachern <dougm at hyperic.com>
+ *   Florian octo Forster <octo at collectd.org>
+ *   Wilfried dothebart Goesgens <dothebart at citadel.org>
+ **/
+
+#include "common.h"
```

Comment:
1.  `"collectd.h"` needs to be the first header included. You will need to add an empty line after it to keep `clang-format` from sorting includes alphabetically.
2.  Please run `clang-format -style=file -i src/curl_jolokia.c`